t/apicast-policy-rewrite-url-captures.t

```diff
@@ -233,3 +233,51 @@ yay, api backend
 --- error_code: 200
 --- no_error_log
 [error]
+
+=== TEST 5: one transformation with method that doesn't match
```

Comment:
We should add two test that match at least one method:

Test6: One single method works. 
Test7: Array with methods [POST, UPDATE, GET] will match if GET method is in there. (Want to check the next method of the array is working)